crates/bevy_math/src/aspect_ratio/mod.rs

```diff
@@ -0,0 +1,108 @@
+//! Provides a simple aspect ratio struct to help with calculations.
+
+pub mod error;
+
+use crate::{aspect_ratio::error::AspectRatioError, Vec2};
+
+#[cfg(feature = "bevy_reflect")]
+use bevy_reflect::Reflect;
+
+/// An `AspectRatio` is the ratio of width to height.
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+#[cfg_attr(feature = "bevy_reflect", derive(Reflect), reflect(Debug, PartialEq))]
+pub struct AspectRatio(f32);
```

Comment:
I think we're missing a getter for the internal field here. You can call .into, but that's pretty obscure.

Reply:
Should the getter be named as_f32 or ratio?
I'm thinking about this, ^-^

Reply:
In my opinion, `ratio` is more appropriate.